Gems/GradientSignal/Code/Source/Components/ImageGradientComponent.cpp

```diff
@@ -550,6 +579,56 @@ namespace GradientSignal
         SetupMultiplierAndOffset(m_configuration.m_scaleRangeMin, m_configuration.m_scaleRangeMax);
     }
 
+    float ImageGradientComponent::GetValueForSamplingType(AZ::u32 x, AZ::u32 y, float pixelX, float pixelY) const
```

Comment:
The parameters doesn't match the illustration.
I think they should be (AZ::u32 x0, AZ::u32 y0, float x, float y)

Reply:
Ah good catch, I added the illustration after I wrote the signature. Will update!